libs/perun/components/src/lib/perun-header/perun-header.component.ts

```diff
@@ -0,0 +1,123 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import {
+  NotificationStorageService,
+  OtherApplicationsService,
+  StoreService,
+} from '@perun-web-apps/perun/services';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import { TranslateService } from '@ngx-translate/core';
+import { getDefaultDialogConfig } from '@perun-web-apps/perun/utils';
+import { MatDialog } from '@angular/material/dialog';
+import { ShowNotificationHistoryDialogComponent } from '@perun-web-apps/perun/dialogs';
+
+export enum AppType {
+  Admin = 'admin',
+  Profile = 'profile',
+  PwdReset = 'pwdReset',
+  Consolidator = 'consolidator',
+  Linker = 'linker',
+}
+
+@Component({
+  selector: 'perun-web-apps-header',
+  templateUrl: './perun-header.component.html',
+  styleUrls: ['./perun-header.component.scss'],
+})
+export class PerunHeaderComponent implements OnInit {
+  @Output() sidenavToggle = new EventEmitter();
+  @Input() hideToggle = false;
+  @Input() hideLanguageMenu = false;
+  @Input() hideHeaderMenu = false;
+  @Input() hideNotifications = true;
```

Comment:
Perhaps it would be better if these served as `showXYZ`. The reason being in case we add some component that is specific for one app we don't need to hide it on all others just show for one.